### DISCUSSION TITLE
https://theme-hope.vuejs.press/zh/guide/markdown/tasklist.html

### DISCUSSION THREAD

**Discussion by giscus** [vuepress-theme-hope/giscus-discussions#659] [Announcements]
# https://theme-hope.vuejs.press/zh/guide/markdown/tasklist.html

让你的 VuePress 站点中的 Markdown 文件支持任务列表。

https://theme-hope.vuejs.press/zh/guide/markdown/tasklist.html

**Comment by DillonHuang** [comment]
AA
